### PR TITLE
chore(upgrade): upgrade PHP version to 7.3.27, 7.4.15 and 8.0.2 and upgrade tools

### PR DESCRIPTION
**Pull request by bdejacobet**

### CODE REVIEW

**Comment by mazsudo** [comment]
could you update changelog too plz @bdejacobet ?

**Comment by bdejacobet** [comment]
@mazsudo I updated the changelog (I fix the previous date too)

**Comment by bdejacobet** [comment]
@mazsudo I upgraded PHP 7.3 and 7.4 and their tools too.

**Comment by bdejacobet** [comment]
@mazsudo Since I upgraded xdebug to version 3, I updated the xdebug.ini to add `xdebug.mode=coverage` by default (Not for PHP7.2 that uses xdebug version 2).

cf. https://xdebug.org/docs/upgrade_guide#New-Concepts

File: php/Dockerfile.j2 [hunk 1]
```diff
@@ -68,7 +68,11 @@ zend_extension=opcache.so \n\
     echo "extension=redis.so" > /usr/local/etc/php/conf.d/redis.ini && \
     curl -sSL https://github.com/xdebug/xdebug/archive/${XDEBUG_VERSION}.tar.gz | tar xz -C /tmp && \
     cd /tmp/xdebug-${XDEBUG_VERSION} && phpize && ./configure --enable-xdebug && make && make install && \
-    echo "zend_extension=xdebug.so" > /usr/local/etc/php/conf.d/xdebug.ini && \
+    if [ "$version" = "72" ]; then \
```
Comment (Bladrak) [inline php/Dockerfile.j2:71]: What for versions before 7.2? Not sure the check is the right one, it it?

**Review by Bladrak — APPROVED** [review]

**Review by Bladrak — APPROVED** [review]

**Pull request merged**